refinery/core/api.py

```diff
@@ -2018,6 +2019,14 @@ def obj_create(self, bundle, **kwargs):
         user = bundle.request.user
         data = json.loads(bundle.request.body)
         new_ext_group = ExtendedGroup(name=data['name'])
+
+        new_ext_group.name = new_ext_group.name.strip()
+        try:
+            new_ext_group.full_clean()
```

Comment:
OK, that's correct. Now, should we allow group names that contain whitespace chars only? What about leading and/or trailing whitespace chars?